packages/adapters/src/llm/pi-ai-adapter.ts

```
@@ -21,6 +21,41 @@ import type {
   ToolDefinition,
 } from "./adapter";
 
+
+const PI_AI_PROVIDER_CANDIDATES = [
```

ESLint

[error] Delete `⏎`

(prettier/prettier)